src/components/common/Header.astro

```diff
@@ -13,17 +14,25 @@ const {
   image,
   headingColor,
   subheadingColor,
+  isContain,
 } = Astro.props;
 ---
 
-<main
-  style={{
+<!--  style={{
     backgroundImage: `url(${image})`,
     backgroundSize: "cover",
     backgroundRepeat: "no-repeat",
     backgroundPosition: "center",
+  }} -->
+<main
+  style={{
+    backgroundImage: `url(${image})`,
   }}
-  class="m-auto flex min-h-[90vh] flex-col items-center justify-between bg-black px-5 pb-12 pt-32 md:rounded-lg"
+  class={`bg-no-repeat ${
+    isContain
+      ? "bg-contain min-h-[35vh] md:min-h-[45vh] lg:min-h-[90vh] md:mt-5 rounded-lg lg:bg-contain"
+      : "bg-cover min-h-[90vh] bg-black  px-5 pb-12 pt-32"
+  } bg-center m-auto flex  flex-col items-center justify-between   md:rounded-lg lg:bg-cover`}
```

Comment:
This is too messy, i suggest to revert back to old and removing header when necessary

Reply:
remove header means? the header contains the image, so removing header will mean removing the image too. and we cannot do dynamic cover or contain previously, so we have to make a new component for these pages only. should i do that? else i cannot think of a way to make only these images "contain" and leave the rest to "cover"

Reply:
Yes correct new component but should be pretty simple as its just a img tag